test/Generic4626Wrapper.t.sol

```diff
@@ -0,0 +1,267 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+import 'forge-std/console2.sol';
+
+import {IDefaultInterestRateStrategyV2} from 'aave-v3-core/contracts/interfaces/IDefaultInterestRateStrategyV2.sol';
+import {IAaveOracle} from 'aave-v3-core/contracts/interfaces/IAaveOracle.sol';
+import {IPool} from 'aave-v3-core/contracts/interfaces/IPool.sol';
+import {IPoolConfigurator} from 'aave-v3-core/contracts/interfaces/IPoolConfigurator.sol';
+import {ConfiguratorInputTypes} from 'aave-v3-core/contracts/protocol/libraries/types/ConfiguratorInputTypes.sol';
+import {IAToken} from 'aave-v3-core/contracts/interfaces/IAToken.sol';
+import {IERC20} from '../src/dependencies/IERC20.sol';
+import {Generic4626Wrapper} from '../src/Generic4626Wrapper.sol';
+import {ICreditDelegationToken} from '../src/interfaces/ICreditDelegationToken.sol';
+import {IBaseTokenWrapper} from '../src/interfaces/IBaseTokenWrapper.sol';
+import {MockERC4626} from './mocks/MockERC4626.sol';
+import {MockERC20} from './mocks/MockERC20.sol';
+import {SigUtils} from './utils/SigUtils.sol';
+import {BaseTokenWrapperTest} from './BaseTokenWrapper.t.sol';
+
+contract Generic4626WrapperTest is BaseTokenWrapperTest {
+  address constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+  address constant POOL_CONFIGURATOR =
+    0x64b761D848206f447Fe2dd461b0c635Ec39EbB27;
+  address constant ADMIN = 0x5300A1a15135EA4dc7aD5a167152C01EFc9b192A;
+  address constant AAVE_ORACLE = 0x54586bE62E3c3580375aE3723C145253060Ca0C2;
+  MockERC20 USDSToken;
+  MockERC4626 SUSDSToken;
+  address USDS;
+  address SUSDS;
+
```

Comment:
better to rename

Reply:
Done [f8b02c0](https://github.com/aave/token-wrappers/commit/f8b02c016564751f9c40f3e236bd3bee738b94d5)